src/Legacy/Numbers/Words/Locale/Es.php

```diff
@@ -77,6 +77,7 @@ class Es extends Words
         'MTL' => [['lira maltesa'], ['céntimo']],
         'MXN' => [['peso'], ['centavo']],
         'NOK' => [['norwegian krone'], ['oere']],
+        'PAB' => [['balboa', 'balboas'], ['centavo', 'centavos']],
```

Comment:
All other currencies have only one version of subcurrency. Are you sure two versions (`'centavo', 'centavos'`) will be handled correctly?

Reply:
I have not tested myself, but for what I understand that array is [singular, plural], not sure why other currencies as MXN which is also from a Spanish speaking country is not handling plural.

Reply:
Also, NOK text is in English in a Spanish file, look at Wikipedia. [https://es.wikipedia.org/wiki/Corona_noruega](url)